### PR TITLE
sys-devel/dwz: Implement "test" USE flag

### PR DESCRIPTION
**Pull request by annulen**

### CODE REVIEW

File: sys-devel/dwz/dwz-0.13.ebuild [hunk 1]
```diff
@@ -14,8 +14,13 @@ S="${WORKDIR}/${PN}"
 LICENSE="GPL-2+ GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
 
-DEPEND="dev-libs/elfutils"
+DEPEND="
+	dev-libs/elfutils
+	test? ( dev-util/dejagnu )
```
Comment (ConiKost) [inline sys-devel/dwz/dwz-0.13.ebuild:22]: Since we are using ```EAPI=7```, this must go into ```BDEPEND```

**Review by ConiKost — CHANGES_REQUESTED** [review]

**Comment by ConiKost** [comment]
And please add a "Closes: https://bugs.gentoo.org/744247" to your commit message, since this is your bug, which you are fixing.

**Comment by annulen** [comment]
Done.

**Review by ConiKost — CHANGES_REQUESTED** [review]
I don't think, that bug ```123456``` is correct :P

File: sys-devel/dwz/dwz-0.13.ebuild [hunk 2]
```diff
@@ -14,8 +14,13 @@ S="${WORKDIR}/${PN}"
 LICENSE="GPL-2+ GPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
 
-DEPEND="dev-libs/elfutils"
+BDEPEND="
+	dev-libs/elfutils
```
Comment (ConiKost) [inline sys-devel/dwz/dwz-0.13.ebuild:21]: And I haven't said, that ```dev-libs/elfutils``` should go into ```BDEPEND``` ;-)

**Comment by annulen** [comment]
> I don't think, that bug `123456` is correct :P

Holy cow!

**Comment by annulen** [comment]
Done.

**Comment by ConiKost** [comment]
I did now give a try, but tests are still failing for me:
```
Running target unix
Using /usr/share/dejagnu/baseboards/unix.exp as board description file for target.
Using /usr/share/dejagnu/config/unix.exp as generic interface file for target.
Using /var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/config/default.exp as tool-and-target-specific interface file.
Running /var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/dwz-tests.exp ...
/var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/pr24173.sh: line 3: eu-strip: command not found
FAIL: /var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/pr24173.sh
/var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/eu-strip-unstrip.sh: line 3: eu-strip: command not found
FAIL: /var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/eu-strip-unstrip.sh
/var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/eu-strip-unstrip-multifile.sh: line 4: eu-strip: command not found
FAIL: /var/tmp/portage/sys-devel/dwz-0.13/work/dwz/testsuite/dwz.tests/eu-strip-unstrip-multifile.sh
```

After recompiling ```dev-libs/elfutils``` with ```USE="utils"``` all tests passed. Please update the dependencies and we should be ready.

**Comment by annulen** [comment]
I think this should make it.

**Review by ConiKost — APPROVED** [review]

**Pull request closed without merging**